modules/install-sno-generating-the-install-iso-manually.adoc

```diff
@@ -12,6 +12,11 @@ Installing {product-title} on a single node requires an installation ISO, which
 
 * Install `podman`.
 
+[NOTE]
+====
+See link:https://docs.openshift.com/container-platform/{latest}/installing/installing_sno/install-sno-preparing-to-install-sno.html[Requirements for installing OpenShift on a single node] for networking requirements, including DNS records.
```

Comment:
No links in a module. This one isn't working in the preview. 

```suggestion
See Requirements for installing OpenShift on a single node for networking requirements, including DNS records.
```
And add an additional reference.

Reply:
Updated As Suggested.